cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/sync/class-upload-sync.php

```diff
@@ -111,15 +117,15 @@ function add_inline_action( $actions, $post ) {
 					'<a href="%s" aria-label="%s">%s</a>',
 					$action_url,
 					/* translators: %s: Attachment title. */
-					esc_attr( sprintf( __( 'Push to Cloudinary &#8220;%s&#8221;' ), 'asd' ) ),
+					esc_attr( __( 'Push to Cloudinary' ) ),
```

Comment:
Couldn't we use `esc_attr__()` instead? Can we add here the plugin namespace for the translation?